src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorEngine.java

```diff
@@ -505,6 +527,11 @@ else if ( funcotatorArgs.referenceVersion.equals(BaseFuncotatorArgumentCollectio
                     "There MAY be some errors (e.g. in the Y chromosome, but possibly in other places as well) due to changes between the two references.");
         }
 
+        // Record whether we need to revert the contigs back to B37 after annotation:
+        if (FuncotatorUtils.isSequenceDictionaryUsingB37Reference(sequenceDictionaryForDrivingVariants) && mustConvertInputContigsToHg19) {
```

Comment:
Since this doesn't affect MAF output you _could_ update this to only run if you're outputting a VCF.  Might save some time.